source/guide_postgresql.rst

```diff
@@ -120,6 +120,8 @@ We now use the temporary password file ``~/pgpass.temp`` (containing only your p
 .. code-block:: console
     :emphasize-lines: 1
 
+    [isabell@stardust ~]$ export LC_CTYPE="de_DE.UTF-8"
```

Comment:
I'm not sure de_DE is the best choice here. The lab is completely in English, exporting a de local would cause some applications to print German instead of English info and error messages.